.stylish-haskell.yaml

```diff
@@ -0,0 +1,63 @@
+steps:
```

Comment:
Why is this file so big? I thought we were just using the defaults except for the `language_prefix`.

Reply:
Apparently you need to provide the whole YAML file from which stylish-haskell needs to read 😞